homeassistant/components/axis/entity.py

```diff
@@ -42,7 +42,8 @@ def __init__(self, device: AxisNetworkDevice) -> None:
         self.device = device
 
         self._attr_device_info = DeviceInfo(
-            identifiers={(AXIS_DOMAIN, device.unique_id)}
+            identifiers={(AXIS_DOMAIN, device.unique_id)},
+            serial_number=device.api.vapix.serial_number,
```

Comment:
```suggestion
            serial_number=device.unique_id,
```
Isn't it better to just use same source?

Reply:
Oh I did not know that was the same. I just looked at some test fixture and found out this was an way to get the value